NuGet.Config

```diff
@@ -5,6 +5,7 @@
     <!-- Do not add any additional feeds if new packages are needed they need to come from nuget.org or our azure-sdk-for-net DevOps feed -->
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="azure-sdk-for-net" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" />
+    <add key="azure_app_service_staging" value="https://www.myget.org/F/azure-appservice-staging/api/v2" />
```

Comment:
This is temporary until new version of WebJobs SDK is available in Nuget.

Reply:
I assume by temporary you don't need to merge this PR in with this new feed correct?

Reply:
Correct

Reply:
WebJobs 3.0.36 has been officially released, you should be able to remove this reference and bump the WebJobs version:

https://www.nuget.org/packages/Microsoft.Azure.WebJobs/3.0.36